### PR TITLE
Adding confirmation page for fact check

### PR DESCRIPTION
**Pull request by LesWilson**
Fact Check confirmation page

### CODE REVIEW

File: app/controllers/FactCheckConfirmController.scala [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+) extends FrontendController(mcc)
+    with I18nSupport {
+
+  val logger: Logger = Logger(getClass)
+
+  def onConfirm(processId: String): Action[AnyContent] = factCheckIdentifierAction.async { implicit request =>
+    reviewService.approvalFactCheckComplete(processId, request.credId, request.name, WithDesignerForUpdate).map {
+      case Right(_) => Ok(view())
+      case Left(NotFoundError) =>
+        logger.error(s"Unable to retrieve approval fact check for process $processId")
```
Comment (woollett-r) [inline app/controllers/FactCheckConfirmController.scala:49]: Possibly worth mentioning confirm action for context.

File: app/views/fact_check_complete.scala.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(layout: main_layout, govukTable : GovukTable)
```
Comment (woollett-r) [inline app/views/fact_check_complete.scala.html:21]: I don't think you need the Gov uk table component if you don't use a table.

File: app/views/fact_check_complete.scala.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(messages("factCheckComplete.heading")) {
```
Comment (woollett-r) [inline app/views/fact_check_complete.scala.html:25]: I would use factCheckComplete.title here.

File: app/controllers/FactCheckConfirmController.scala (hunk 1 above)
Comment (LesWilson) [reply app/controllers/FactCheckConfirmController.scala:49]: done

File: app/views/fact_check_complete.scala.html (hunk 2 above)
Comment (LesWilson) [reply app/views/fact_check_complete.scala.html:21]: removed

File: app/views/fact_check_complete.scala.html (hunk 3 above)
Comment (LesWilson) [reply app/views/fact_check_complete.scala.html:25]: done by Chris

**Pull request merged**